<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabSfx</class>
 <widget class="QWidget" name="TabSfx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select Entry:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="select_comboBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="add_pushButton">
     <property name="text">
      <string>Add Entry</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="name_lineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="filename_comboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
